boa3_test/tests/test_classes/testengine.py

```diff
@@ -10,11 +10,16 @@
 from boa3_test.tests.test_classes.block import Block
 from boa3_test.tests.test_classes.storage import Storage
 from boa3_test.tests.test_classes.transaction import Transaction
+from os import path
 
 
 class TestEngine:
     def __init__(self, root_path: str):
-        self._test_engine_path = '{0}/Neo.TestEngine/Neo.TestEngine.dll'.format(root_path)
+        if path.exists('{0}/Neo.TestEngine/Neo.TestEngine.dll'.format(root_path)):
+            self._test_engine_path = '{0}/Neo.TestEngine/Neo.TestEngine.dll'.format(root_path)
+        else:
+            raise Exception("Can not open file 'Neo.TestEngine/Neo.TestEngine.dll'. No such file exists."
```

Comment:
Show the given path in the exception
Also, use a specific exception for this. It can be Python's built-in `FileNotFoundError`.